cpp/src/parquet/metadata.h

```diff
@@ -157,7 +157,6 @@ class PARQUET_EXPORT ColumnChunkMetaData {
   const std::string& file_path() const;
 
   // column metadata
-  bool is_metadata_set() const;
```

Comment:
This method is not implemented and not been used, I don't know why this guy is here...